src/components/atoms/buttons/HireMeButton.tsx

```diff
@@ -0,0 +1,36 @@
+import { Button } from '@mui/material'
+import { Link, useLocation } from 'react-router-dom'
+
+export function HireMeButton() {
+  const location = useLocation()
+  const isHomePage = location.pathname === '/'
+
+  if (isHomePage) {
+    return null
+  }
```

Comment:
🔧  I don't like the idea of an atom making this decision. Whether a component is rendered or not should come from its parent.

Reply:
Ok, I'll move this into a Molecule.

Reply:
Handling it like this makes more sense and is way more clean.